plugin/src/main/java/io/spine/tools/gradle/bootstrap/Extension.java

```diff
@@ -117,6 +136,51 @@ public JavaScriptExtension enableJavaScript() {
      */
     void disableJavaGeneration() {
         java.disableGeneration();
+        toggleJavaTasks(false);
+    }
+
+    private void toggleJavaTasks(boolean shouldEnable) {
+        this.javaEnabled = shouldEnable;
+        toggleCompileJavaTasks(shouldEnable);
+        toggleTransitiveProtos(shouldEnable);
+    }
+
+    /**
+     * If the {@code protobuf} configuration is present, toggles the transitivity according to the
+     * specified parameter.
+     *
+     * <p>Disabling transitivity leads to exclusion of {@code spine} and
+     * {@code com.google.protobuf} dependencies.
+     *
+     * @param shouldEnable whether the transitivity should be enabled
+     */
+    private void toggleTransitiveProtos(boolean shouldEnable) {
+        project.configurations(closure((ConfigurationContainer container) -> {
+            try {
+                @SuppressWarnings("DuplicateStringLiteralInspection" /* Different context. */)
+                Configuration protobuf = container.getByName("protobuf");
```

Comment:
There must be a constant in `CnfigurationName` in `base`. If it's not there, please add it.